### PR TITLE
Adds new pickaxe types

### PR DESCRIPTION
**Pull request by ShizCalev**
Adds more picks
Each pick is a upgrade to the last
Updated the cost of the picks in the vender to reflect that they're upgrades
Ultimately this gives the diamond pick more value as a boss drop item (which it is)

### CODE REVIEW

**Comment by bgobandit** [comment]
Mats 5 was for a reason, you can get the pickaxe for relatively few points out of the vendor and that makes R&D too easy

**Comment by ShizCalev** [comment]
Fair enough.

**Comment by FantasticFwoosh** [comment]
👎 from me, this is unnessecary bloat and all of these besides the uranium pick are functionally the same and effectively worthless to add in because picks beside the top tier diamond pick are effectively worthless.

**Comment by Bawhoppen** [comment]
bloat is a meme and variety is good however the balance concerns related do still matter

**Comment by ShizCalev** [comment]
Added a new stat to picks, durability. Basic minecraft mode, picks break after a certain number of uses. 

Numbers are subject to change, feel free to recommend different values.

**Comment by ChangelingRain** [comment]
No durability. Just don't do that, it sucks and will only annoy people given that miners have KAs and etc.

**Comment by GunHog** [comment]
Nerfing an item no one even uses except Gulag slaves (And Dwarves, but those do not exist yet)?

**Comment by Hatfish** [comment]
This is a pretty heavy nerf to all lavaland roles, considering that most of them will more often than not start surrounded by rock and have to dig to reach new cave systems.

**Comment by ShizCalev** [comment]
@Hatfish As I said, values are subject to change. Currently you can travel 100 squares in any direction with the basic pickaxe before having to head back home to grab another, which is able to get you halfway from one end of lavaplanet to the other if you're digging a straight line through rock.

**Comment by ChangelingRain** [comment]
Just don't do durability at all.

**Comment by ShizCalev** [comment]
Fiiiiiiiiine

**Comment by ShizCalev** [comment]
Tweaked the costs of the picks. Gold is faster than silver. Plasma is faster than gold (they take the spot diamond picks used to hold.) Diamond picks are now slightly faster. Uranium picks are the slowest out of all of them, but have more force.

**Comment by dannno** [comment]
bloat is not a meme
I don't see the point to this

**Comment by DoctoreMad** [comment]
This is double the bloat when you consider dwarves will be making silver and plasma pickaxes too. So you have two types of pickaxe, one is form the mining vendor, other is form dwarves, they have different stats because ???.

**Comment by Ergovisavi** [comment]
I am seriously struggling to understand the purpose of this, and the mining vendor's """""UI""""" is already filled to the brim and unsorted, this won't help that

**Comment by ShizCalev** [comment]
@DoctoreMad i didn't add silver pickaxes, those are already ingame fam.

**Comment by ShizCalev** [comment]
@Ergovisavi all the picks are groups together with this, which is better than how it currently is.

File: code/modules/mining/equipment/mining_tools.dm [hunk 1]
```diff
@@ -37,13 +37,40 @@
...
+	origin_tech = "materials=2;engineering=3;plasmatech=2"
+	desc = "A pickaxe with a solidified plasma pick head. Extremely robust at cracking rock walls and digging up dirt."
+	force = 17
+
 /obj/item/weapon/pickaxe/diamond
 	name = "diamond-tipped pickaxe"
 	icon_state = "dpickaxe"
 	item_state = "dpickaxe"
-	digspeed = 14
+	digspeed = 11
```
Comment (Jalleo) [inline code/modules/mining/equipment/mining_tools.dm]: Nerfing the diamond one I see. Why?

Comment (XDTM) [inline code/modules/mining/equipment/mining_tools.dm]: i think that's the time needed to dig, so it'd be a buff

**Comment by FantasticFwoosh** [comment]
Objectively none of this also matter in the case of if you have ANY access to RND you can print out a more superior version of manual pickaxes from the vendor.

- Diamond Tipped drill

- Sonic Jackhammer (faster than that diamond upgrade)

Along with other items such as KA & resonator upgrades that are marginally more effective at what has been set out. The OP is clarifying they are doing it "just because the sprites are there" tried & failed to add more goals onto the brief (durability ☠️ ) and has changed it halfway through to alter the items it was affecting.

File: code/modules/mining/equipment/mining_tools.dm (hunk 1 above)
Comment (ShizCalev) [inline code/modules/mining/equipment/mining_tools.dm]: Plasma pick has the stats of the current diamond pick, and the diamond pick was made better (faster), giving it more value as an upgrade / item drop.

**Comment by dannno** [comment]
"just because the sprites are there" is not such a good reason

**Comment by ShizCalev** [comment]
@dannno @FantasticFwoosh I changed the op's description a while ago yo.

also @FantasticFwoosh, why would the diamond drill get a buff? It's still twice as fast as the top tier pick & is a direct upgrade over all of them. Currently on live, the silver pickaxe & diamond pickaxe are both faster than a R&D produced drill; should all three of them get removed because R&D can make diamond drills that make all three items obsolete by the end of the round?

Imo using the line "r&d can make better shit" is pretty crappy. Why stunt giving a department more options for equipment upgrades independent of R&D because R&D has the ability to produce superior end-game versions of everything under the sun?

**Comment by dannno** [comment]
you didn't change it before we commented

**Comment by ShizCalev** [comment]
@dannno Yes, that comment was already changed when I did the durability commit yesterday.

**Comment by FantasticFwoosh** [comment]
Just because the system is flawed with RnD having better equipment doesn't justify this PR existing despite your argument that the powerful items shouldn't all be present inside RnD which im generally in line to agree with.

- I mention the diamond drill in relevancy to the existing diamond pick because they are essentially the same object in terms of power & handling, your

**Comment by octareenroon91** [comment]
You want to use sprites? Give an existing pick the reskin features.

You want to make a serious upgrade? Take the moderately-faster, heavy-hitting uranium pickaxe concept and instead add an upgrade to the mini pickaxe.

**Comment by FantasticFwoosh** [comment]
Just providing detachable pick head mods (sort of but not actually dwarves) would have been a better alternative following on from @octareenroon91 comment, because in itself the item is worthless to miners and there's better combat weapons in a bone axe rather than a uranium pickaxe for improv miner gear.

**Comment by WJohn** [comment]
what's the point though?

everyone's still gonna use the KA and plasma cutter, because picks suck ass.

**Comment by octareenroon91** [comment]
>everyone's still gonna use the KA and plasma cutter, because picks suck ass.

Picks are OK because you can just mine with the keyboard.

**Comment by optimumtact** [comment]
just remove picks they're so pointless, they can be added back for dwarves but they literally have no value in game.

Give everyone a basic drill to spawn with instead.

**Pull request closed without merging**